### DISCUSSION TITLE
Updating mpv inside smplayer portable

### DISCUSSION THREAD

**Discussion by Raza-0007** [smplayer-dev/smplayer#697] [General]
I have a question for the developers:

I noticed that inside smplayer portable folder, there is a sub-folder labeled "mpv". Inside this mpv sub-folder there are 5 items; two folders labeled "Doc" and "mpv" and three files "d3dcompiler_43.dll", "mpv.com" and "mpv.exe".

These appear to be the same 5 items present inside the folder of a separately downloaded portable mpv.

My question is, if I replace these 5 items inside smplayer\mpv\ with the contents of a newly downloaded portable mpv, will it update the version of mpv inside smplayer to the latest 0.35.1 without breaking anything? Is this a recommended way of updating the mpv engine inside smplayer? or should I just wait for the developers to release a new smplayer with updated mpv inside it?

Thanks,

Raza

**Comment by smplayer-dev** [comment]
Usually there shouldn't be any problem updating mpv.
You can also install mpv in a different folder and set the mpv.exe path in the smplayer settings.

**Reply by Raza-0007** [reply]
Thanks for the response.

After updating mpv to 0.35.1, I noticed a slight stutter in video playback that does not occur when using the mpv version bundled in with smplayer. However, it might just be my imagination. I will test it further.

Any chance of you guys releasing an official build with the updated mpv? One should be due soon seeing that the last official build was in July 2022.

Anyway, keep up the good work. Smplayer has been my default media player for the last 3 years, and the only mpv front-end I prefer.

**Reply by Raza-0007** [reply]
I just wanted to report back after doing further tests. The slight video stutter I noticed after updating mpv inside smplayer portable was most likely my imagination, because I have not noticed it since then. All videos play fine. However, I have not noticed any real difference in video/audio quality between the mpv 0.34 that is bundled in and the latest mpv 0.35.1.
